ambry-commons/src/test/java/com.github.ambry.commons/HelixNotifierTest.java

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.commons;
+
+import com.github.ambry.config.HelixPropertyStoreConfig;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicReference;
+import org.apache.helix.ZNRecord;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static com.github.ambry.commons.TestUtils.*;
+import static org.junit.Assert.*;
+
+
+/**
+ * Unit tests for {@link HelixNotifier}.
+ */
+public class HelixNotifierTest {
+  private static int zkClientConnectTimeoutMs = 20000;
```

Comment:
these can all be `final`? also `static final` variables should be in all caps and separated by underscore (this will be `ZK_CLIENT_CONNECT_TIMEOUT_MS`) - all `static` variables here can be made `final` i think